aws_jobs/cyhy-data-extract.py

```diff
@@ -328,63 +334,118 @@ def main():
+                "retired": True,
+                "scan_types": True,
+                "stakeholder": True,
             },
         },
         "tickets": {
-            "owner": {"$in": orgs},
-            "last_change": {
-                "$gte": start_of_data_collection,
-                "$lt": end_of_data_collection,
+            "query": {
+                "owner": {"$in": orgs},
+                "last_change": {
+                    "$gte": start_of_data_collection,
+                    "$lt": end_of_data_collection,
+                },
+            },
+            "projection": default_projection,
+        },
+        "vuln_scans": {
+            "query": {
+                "owner": {"$in": orgs},
+                "time": {
+                    "$gte": start_of_data_collection,
+                    "$lt": end_of_data_collection,
+                },
             },
+            "projection": default_projection,
         },
     }
 
     scan_collection = {
+        "certs": {
+            "query": {
+                "sct_or_not_before": {
+                    "$gte": start_of_data_collection,
+                    "$lt": end_of_data_collection,
+                }
+            },
+            "projection": default_projection,
+        },
         "https_scan": {
-            "scan_date": {
-                "$gte": start_of_data_collection,
-                "$lt": end_of_data_collection,
-            }
+            "query": {
+                "scan_date": {
+                    "$gte": start_of_data_collection,
+                    "$lt": end_of_data_collection,
+                }
+            },
+            "projection": default_projection,
         },
         "sslyze_scan": {
-            "scan_date": {
-                "$gte": start_of_data_collection,
-                "$lt": end_of_data_collection,
-            }
+            "query": {
+                "scan_date": {
+                    "$gte": start_of_data_collection,
+                    "$lt": end_of_data_collection,
+                }
+            },
+            "projection": default_projection,
         },
         "trustymail": {
-            "scan_date": {
-                "$gte": start_of_data_collection,
-                "$lt": end_of_data_collection,
-            }
-        },
-        "certs": {
-            "sct_or_not_before": {
-                "$gte": start_of_data_collection,
-                "$lt": end_of_data_collection,
-            }
+            "query": {
+                "scan_date": {
+                    "$gte": start_of_data_collection,
+                    "$lt": end_of_data_collection,
+                }
+            },
+            "projection": default_projection,
         },
     }
 
-    assessment_collection = {"assessments": {}, "findings": {}}
+    assessment_collection = {
+        "assessments": {"query": {}, "projection": default_projection},
+        "findings": {"query": {}, "projection": default_projection},
+    }
```

Comment:
@mcdonnnj - Same as above, I think we should add a brief comment here explaining why every document in the `assessments` and `findings` collections is being exported, rather than a subset based on a date field.

Reply:
Good thought, I added an informative comment in 401fae5